Code/Tools/ProjectManager/Source/GemCatalog/GemCatalogHeaderWidget.cpp

```diff
@@ -288,29 +289,40 @@ namespace O3DE::ProjectManager
         }
     }
 
-    void CartOverlayWidget::GemDownloadProgress(const QString& gemName, int percentage)
+    void CartOverlayWidget::GemDownloadProgress(const QString& gemName, int bytesDownloaded, int totalBytes)
     {
         QWidget* gemToUpdate = m_downloadingListWidget->findChild<QWidget*>(gemName);
         if (gemToUpdate)
         {
             QLabel* progressLabel = gemToUpdate->findChild<QLabel*>("DownloadProgressLabel");
-            if (progressLabel)
+            QProgressBar* progressBar = gemToUpdate->findChild<QProgressBar*>("DownloadProgressBar");
+
+            if (totalBytes != 0)
```

Comment:
hmmm I guess it's not in the design but as a user, I'd like to see the bytes downloaded, total bytes and percentage if available.  I guess the space is too small to show that?

Reply:
Yes, there would probably need to be a new line to contain all the information - I'll bring this up with UX again once this change is in. Seeing the bytes downloaded for the servers without content length felt more useful than even seeing the text percentage, it should be possible to move things around a bit to show everything though.